### PR TITLE
Add Anchorage node operation

### PR DESCRIPTION
**Pull request by boyuan-chen**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Add `node operators` pages for Anchorage

### CODE REVIEW

**Review by wsdt — APPROVED** [review]

**Review by souradeep-das — APPROVED** [review]
🚀

File: SUMMARY.md [hunk 1]
```diff
@@ -21,10 +21,16 @@
...
   * [xDomain Transaction Status](boba\_documentation/developer/xdomain-tx-status.md)
-  * [Replica node](boba\_community/boba-node/README.md)
   * [Exchange Integration](boba\_documentation/developer/exchange-integration.md)
   * [Contracts Registration](for-developers/register.md)
   * [Token Addresses](boba\_documentation/developer/token-addresses.md)
+  * [Node Operators](for-developers/node-operators/README.md)
+    * [Overview](for-developers/node-operators/overview.md)
+    * [Run a Node With Docker](for-developers/node-operators/run-node-docker.md)
+    * [Run a Node From Source](for-developers/node-operators/run-node-source.md)
+    * [snapshot downloads](for-developers/node-operators/snapshot-downloads.md)
```
Comment (souradeep-das) [inline SUMMARY.md]: minor: 
```suggestion
    * [Snapshot downloads](for-developers/node-operators/snapshot-downloads.md)
```

File: SUMMARY.md [hunk 2]
```diff
@@ -21,10 +21,16 @@
...
+    * [software release](for-developers/node-operators/software-release.md)
```
Comment (souradeep-das) [inline SUMMARY.md]: minor: 
```suggestion
    * [Software release](for-developers/node-operators/software-release.md)
```

File: for-developers/node-operators/overview.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+The Rollup Node is responsible for taking data from Layer 1 (L1) and use it to create Layer 2 (L2) block information, which it then sends to the Execution Client. Additionally, the Rollup Node can choose to join a network that lets it get blocks straight from the Sequencer before those blocks are sent to L1.
+
+### Execution Client
+
+The Execution Client takes care of running the block information it gets from the Rollup Node. It does this through a common method called JSON-RPC, using a set of rules known as the [Engine API](https://github.com/ethereum/execution-apis/blob/main/src/engine/common.md#engine-api----common-definitions). Boba allows the use of either geth or erigon as the Execution Client.
+
+### Legacy Geth
+
+Boba switched its database from **Geth** to **Erigon** during a big update called the [Anchorage Upgrade](https://github.com/bobanetwork/v3-anchorage/blob/develop/boba-chain-ops/README.md#migration). The old data from Geth, before this upgrade, can still be accessed with the new system but needs a special part called Legacy Geth to work properly. This means if you need to use certain commands, like `eth_call`, on the old data, you'll have to use Legacy Geth.
```
Comment (jyellick) [inline for-developers/node-operators/overview.md]: Nit: s/big/significant/

**Review by jyellick — APPROVED** [review]

**Pull request merged**